lms/static/sass/shared-v2/_course-outline.scss

```diff
@@ -0,0 +1,45 @@
+.course-outline {
+  color: $lms-gray;
+
+  ol {
```

Comment:
NIT: Do we still generally avoid element selectors to have class based selection? I ask because of how many problems we've had in the past with how general our early scss was.